daffodil-tdml-processor/src/main/scala/org/apache/daffodil/processor/tdml/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -334,7 +338,9 @@ class DaffodilTDMLDFDLProcessor private (private var dp: DataProcessor)
       val saxInputStream = optSaxInputStream.get
       val saxOutputStream = new ByteArrayOutputStream
       val saxOutputChannel = java.nio.channels.Channels.newChannel(saxOutputStream)
-      val unparseContentHandler = dp.newContentHandlerInstance(saxOutputChannel)
+      val unparseContentHandler = dp
+        .newContentHandlerInstance(saxOutputChannel)
+        .asInstanceOf[DaffodilUnparseContentHandler]
```

Comment:
Is this needed only because we are using 'enableResolutionOfRelativeInfosetBlobs', which is not public? Can you add a comment about that?